lib/macros_autocompletion/macro_completer.js

```diff
@@ -1,5 +1,5 @@
 "use strict"
```

Comment:
For the sake of consistency, please prefer appropriate use of `let` and `const` to `var` and avoid using semi-colons when it is convenient.